frontend/fingertips-frontend/mock/server/handlers.ts

```diff
@@ -87,8 +88,8 @@ export function getGetIndicator500Response() {
   };
 }
 
-export function getGetHealthDataForAnIndicator200Response() {
-  return mockHealthData;
+export function getGetHealthDataForAnIndicator200Response(indicatorId: string) {
+  return mockHealthData[indicatorId];
```

Comment:
This is great, in that it allows us to have different data for a specific indicator, but it does mean that if we give an indicator ID other than `1` or `92708` we don't get any data. The following change would mean that we get that custom indicator data, but we can return default data for any indicator provided, which I think will be much more useful for a mock endpoint:
```suggestion
  if (indicatorId in mockHealthData) {
    return mockHealthData[indicatorId];
  }

  return mockHealthData[1];
```